src/IO/CompressionMethod.cpp

```diff
@@ -55,6 +55,32 @@ std::string toContentEncodingName(CompressionMethod method)
     __builtin_unreachable();
 }
 
+CompressionMethod chooseHTTPCompressionMethod(const std::string & list)
+{
+    /// If client supports brotli - it's preferred.
```

Comment:
The order can be changed: zstd is slightly better than brotli and...
here is the list from the most preferred to the least preferred:

zstd
br
lz4
snappy
gzip
deflate
xz
bz2

Reply:
Done